packages/node_modules/@ciscospark/internal-plugin-conversation/src/conversation.js

```diff
@@ -249,6 +249,10 @@ const Conversation = SparkPlugin.extend({
           }, omit(options, 'id', 'user', 'url'))
         };
 
+        // to prevent a 401, this will add the auth token while doing
+        // key rotation
+        params.runWhitelistedDomains = true;
```

Comment:
1. Can you think of any time when we *wouldn't* want to do this? 
2. Please [add a test](https://github.com/webex/spark-js-sdk/blob/master/packages/node_modules/@ciscospark/internal-plugin-conversation/test/integration/spec/get.js#L75).